src/page-layout/Main/Main.jsx

```diff
@@ -21,10 +24,19 @@ function Main() {
           <Route path="/" element={<Home />} />
           <Route path="/home" element={<Home />} />
```

Comment:
is it good to use two route for same page ?

Reply:
The root route is for landing page, will update it